app/cdap/components/PipelineDetails/RunLevelInfo/index.js

```diff
@@ -43,6 +43,10 @@ const ConnectedRunNumWarnings = connect(mapStateToProps)(RunNumWarnings);
 const ConnectedRunNumErrors = connect(mapStateToProps)(RunNumErrors);
 
 export default function RunLevelInfo() {
+  // error and warning counts are disabled when error classification is available,
+  // i.e. post v6.11.0
+  const shouldShowErrorAndWarningCounts = false;
```

Comment:
What's the condition that enables this flag? I see that it is always false now which means Warnings & Error options will be not be shown. Is that expected ?